crypto/types/multisig/threshold_pubkey.go

```diff
@@ -35,9 +39,12 @@ func NewPubKeyMultisigThreshold(k int, pubkeys []crypto.PubKey) crypto.PubKey {
 // and all signatures are valid. (Not just k of the signatures)
 // The multisig uses a bitarray, so multiple signatures for the same key is not
 // a concern.
+//
+// NOTE: VerifyMultisignature should preferred to VerifyBytes which only works
+// with amino multisignatures.
```

Comment:
Can we deprecate? Or it is not the objective?

Reply:
`VerifyBytes` is a method on the `PubKey` interface so we can't really deprecate it yet. But I do think we want to migrate away from the tendermint interface and have a Cosmos SDK `PubKey` interface that we can evolve.